import_test_data/run_imports.sh

```diff
@@ -0,0 +1,362 @@
+  if [ $? != 0 ];then
+    fail_test "directory import should have succeeded"
+  fi
+}
+
+specify_db_tests() {
+  cleanup_all_collections
+  $FAUNA_CMD import --endpoint data-import-test --path=type_tests --db=Nope
+  if [ $? == 0 ];then
+    fail_test "directory import should have failed as db doesn't exist"
+  fi
+
+  cleanup_collection "json_array"
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_array.json --db=Nope
+  if [ $? == 0 ];then
+      fail_test "json_array.json should have failed to import as db doesn't exist"
+  fi
+
+  cleanup_all_collections
+  ensure_db "test-specify-db"
+  $FAUNA_CMD import --endpoint data-import-test --path=type_tests --db=test-specify-db
+  if [ $? != 0 ];then
+    fail_test "directory should have succeeded"
+  fi
+
+  cleanup_collection "json_array"
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_array.json --db=test-specify-db
+  if [ $? != 0 ];then
+    fail_test "json_array.json should have succeeded"
+  fi
+}
+
+append_tests() {
+  cleanup_collection "json_array"
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_array.json
+  if [ $? != 0 ];then
+      fail_test "json_array.json failed to import initally"
+  fi
+ $FAUNA_CMD import --endpoint data-import-test --path=json/json_array.json
+  if [ $? == 0 ];then
+      fail_test "json_array.json should fail as the append flag was not sent"
+  fi
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_array.json --append
+  if [ $? != 0 ];then
+      fail_test "json_array.json failed to append"
+  fi
+  cleanup_collection "json_l"
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_l.jsonl
+  if [ $? != 0 ];then
+    fail_test "json_l.jsonl failed to import initially"
+  fi
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_l.jsonl
+  if [ $? == 0 ];then
+    fail_test "json_l.jsonl should fail as the append flag was not sent"
+  fi
+  $FAUNA_CMD import --endpoint data-import-test --path=json/json_l.jsonl --append
+  if [ $? != 0 ];then
+    fail_test "json_l.jsonl failed to append"
+  fi
+  cleanup_collection "bool_type"
+  $FAUNA_CMD import --endpoint data-import-test --type=favorite::bool --path=type_tests/bool_type.csv
+  if [ $? != 0 ];then
+    fail_test "bool_type.csv didn't import intially with success"
+  fi
+  $FAUNA_CMD import --endpoint data-import-test --type=favorite::bool --path=type_tests/bool_type.csv
+  if [ $? == 0 ];then
+    fail_test "bool_type.csv should fail as the append flag was not sent"
+  fi
+  $FAUNA_CMD import --endpoint data-import-test --type=favorite::bool --path=type_tests/bool_type.csv --append
+  if [ $? != 0 ];then
+    fail_test "bool_type.csv didn't failed to append"
+  fi
+}
+
+# Comment out test batches as required.
+# run_type_tests
+# short_row_tests
+# header_name_tests
+# json_tests
+# invalid_file_type_tests
+directory_tests
+# append_tests
+# directory_specify_collection_tests
+# specify_db_tests
```

Comment:
Let's leave these uncommented to start. People should run all the tests by default.

Reply:
oops!

Reply:
fixed.